app/src/app/api/v0/inventory/[inventory]/value/subsector/[subsector]/route.ts

```diff
@@ -13,7 +13,15 @@ export const GET = apiHandler(async (_req, { params, session }) => {
       subSectorId: params.subsector,
       inventoryId: inventory.inventoryId,
     },
-    include: [{ model: db.models.DataSource, as: "dataSource" }],
+    include: [
+      { model: db.models.DataSource, as: "dataSource" },
+      {
+        model: db.models.SubCategory,
+        as: "subCategory",
+      },
+      { model: db.models.Sector, as: "sector" },
+      { model: db.models.SubSector, as: "subSector" },
+    ],
```

Comment:
Can you add this to the documentation of the endpoint in the Github wiki API docs?